ComputerInterface/Views/GameSettings/GroupView.cs

```diff
@@ -56,11 +57,10 @@ public void DrawHeader(StringBuilder str)
 
         public void DrawOptions(StringBuilder str)
         {
-            str.Append(_selectionHandler.GetIndicatedText(0, "Forest")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(1, "Cave")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(2, "Canyon")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(3, "City")).AppendLine();
-            str.Append(_selectionHandler.GetIndicatedText(4, "Mountain")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(0, "Forest  ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(1, "Cave    ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(2, "Canyon  ")).AppendLine();
+            str.Append(_selectionHandler.GetIndicatedText(3, "City    ")).AppendLine();
```

Comment:
What happened to mountain?

Reply:
i couldn't get it working.

Reply:
all of them worked fine but when i tried mountain it didnt work + none worked in mountain

Reply:
ill prob try to add it back when i have the time..

Reply:
Fixed in commit 7b314ddc3f71260aa748c6729ec6955d1d22870c